tools/cpp/BUILD.empty.tpl

```diff
@@ -28,8 +29,8 @@ filegroup(
 cc_toolchain_suite(
     name = "toolchain",
     toolchains = {
-        "local|local": ":local",
-        "local": ":local",
```

Comment:
it's unclear to me if the 'local' tag is suppose to have any meaning anymore, I wasn't able to dig into the history, was this the default name of the host configuration before platforms were a thing?